src/brokers/broker-details/components/Queues/Queues.component.tsx

```diff
@@ -0,0 +1,59 @@
+import { useTranslation } from '../../../../i18n';
+import {
+  TableColumn,
+  VirtualizedTable,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { Queue } from './Queues.container';
+import { QueueRow } from './QueueRow';
+
+export type QueuesProps = {
+  queueData: Queue[];
+  isLoaded: boolean;
+  loadError: boolean;
+};
+
+const Queues: React.FC<QueuesProps> = ({ queueData, isLoaded, loadError }) => {
+  const { t } = useTranslation();
+
+  const columns: TableColumn<Queue>[] = [
+    {
+      title: t('Name'),
+      id: 'name',
+    },
+    {
+      title: t('Routing type'),
```

Comment:
```suggestion
      title: t('routing_type'),
```

Please keep translations string key in lower case with snake format.

Reply:
Don't forget to add all the translation string in resource file. 
https://github.com/artemiscloud/activemq-artemis-self-provisioning-plugin/blob/main/locales/en/plugin__activemq-artemis-self-provisioning-plugin.json